docs/DeveloperGuide.md

```diff
@@ -362,55 +363,239 @@ testers are expected to do more *exploratory* testing.
 
 #### **Launch and Shutdown**
 
-1. Initial launch
+#### Scenario 1  
+{: .no_toc}  
+**Context:** Initial launch
 
-   1. Download the jar file and copy into an empty folder
+**Action:**
+1. Download the jar file and copy into an empty folder
+2. Double-click the jar file
 
-   1. Double-click the jar file Expected: Shows the GUI with 3 components (The window size may not be
-      optimum):
-      1. A calendar of the month of present time 
-      2. An input bar for user commands
-      3. A container showing the next upcoming event.
+**Result:**  Displays the GUI with the window size set to full-screen
```

Comment:
Perhaps rephrase to "..with the application opened to full-screen"?

Reply:
Ok